docs/configuring.md

```diff
@@ -385,7 +385,7 @@ running `copier update`, this setting has no effect.
 
 ### `data`
 
--   Format: `dict|List[str=str]`
+-   Format: `dict|List[str=str]|yamlfile`
```

Comment:
Please undo this change, as it is only meant for API AFAICR (you can't pass a `dict` through CLI).